app-center-services/src/main/java/org/exoplatform/appcenter/storage/ApplicationCenterStorage.java

```diff
@@ -303,8 +308,17 @@ private Application toDTO(ApplicationEntity applicationEntity) throws FileStorag
     if (applicationEntity == null) {
       return null;
     }
-    String imageFileName = applicationEntity.getImageFileId() != null
-        && applicationEntity.getImageFileId() > 0 ? fileService.getFile(applicationEntity.getImageFileId()).getFileInfo().getName() : null;
+    String imageFileName = null;
+    FileItem file = null;
+    if(applicationEntity.getImageFileId()>0){
+      try {
+        file = fileService.getFile(applicationEntity.getImageFileId());
+        imageFileName = file.getFileInfo().getName();
```

Comment:
You can use FileService.getFileInfo(fileId) WDYT ?

Reply:
done